bundles/org.openhab.binding.hdpowerview/README.md

```diff
@@ -1,69 +1,196 @@
+- For each scene discovered: the binding will create a new channel dynamically within the hub thing.
+
+If in the future, you add additional shades or scenes to your system, the binding will discover them too.
 
 ## Thing Configuration
 
-PowerView things should be configured via discovery - it would be difficult to configure manually as the IDs of the shades and scenes are not exposed via the dedicated app.
+### Thing Configuration for PowerView Hub
+
+| Configuration Parameter | Description   |
+|-------------------------|---------------|
+| host                    | The host name or IP address of the hub on your network. |
+| refresh                 | The number of milli-seconds between fetches of the PowerView hub's shade state (default 60'000 one minute). |
+
+### Thing Configuration for PowerView Shades
+
+PowerView shades should preferably be configured via the automatic discovery process.
+It is quite difficult to configure manually as the `id` of the shade is not exposed in the PowerView app.
 However, the configuration parameters are described below:
 
-<table>
- <tr>
-  <td><b>Thing</b></td>
-  <td><b>Configuration Parameters</b></td>
- </tr>
- <tr>
-  <td>PowerView Hub</td>
-  <td>
-   <table>
-    <tr><td><b>host</b> - the hostname or IP address of the Hub on your network.</td></tr>
-    <tr><td><b>refresh</b> - the number of milliseconds between fetches of the PowerView Hub's shade state. Defaults to 60,000 (one minute).</td></tr>
-   </table>
-  </td>
- </tr>
- <tr>
-  <td>PowerView Shade</td>
-  <td>
-   <table>
-    <tr><td><b>id</b> - the ID of the PowerView Shade on the Hub.</td></tr>
-   </table>
-  </td>
- </tr>
-</table>
+| Configuration Parameter | Description                                                   |
+|-------------------------|---------------------------------------------------------------|
+| id                      | The ID of the PowerView shade in the app. Must be an integer. |
 
 ## Channels
 
-### PowerView Shade
+### Channels for PowerView Hub
+
+The hub always has one fixed channel as below:
+
+| Channel  | Item Type | Description |
+|----------|-----------| ------------|
+| refresh  | Switch    | If you switch on this channel, it will make the hub scan all shades and update its cache state -- _see "**Refreshing the PowerView Hub Cache**" below_. Note: include `{autoupdate="false"}` in the item configuration to avoid having to reset it to off after use. |
+
+In addition, scene channels will be added dynamically to the binding as they are discovered in the hub.
+Each scene channel will have an entry in the hub as shown below, whereby different scenes have different `id` values:
+
+| Channel  | Item Type | Description |
+|----------|-----------| ------------|
+| id | Switch | Turning this to ON will activate the scene. Scenes are stateless in the PowerView hub; they have no on/off state. Note: include `{autoupdate="false"}` in the item configuration to avoid having to reset it to off after use. |
```

Comment:
There are two reasons for NOT changing this..

1. The prior version of the binding already had Scenes = Channels. So changing the paradigm now would be a **"breaking change"** for existing users.
2. Similar argument concerning UI access to the commands as mentioned in the topic "hard refresh above" (see also the BasicUI screenshot above)

Reply:
I understood that every scene Channel gets an additional Channel in the bridge, which is a new feature. I don't understand why changing this new feature would break the existing Channels.

Can the user set the scene ID to `refresh`?

Reply:
> I understood that every scene Channel gets an additional Channel in the bridge, which is a new feature. 

No. This is not a new feature. The original binding was always that way.

The new feature is that in addition to having one Channel in the hub for each scene, there is now also one more Channel for the hard refresh. I don't want to change the functioning of the Scene Channels, since that would be a breaking change. I would like to keep the added hard refresh function as a Channel since it would function identically to the Scene Channels, making it easier for the average user to understand.

Reply:
Ok, you have your reasons to stick to it. I don't want to push you in any direction.

Reply:
I deleted the hard refresh Channel. And I have left the Scene Channels "not broken" as they were.